app/assets/stylesheets/bootstrap_bundle.scss

```diff
@@ -214,6 +214,9 @@ body > div > .popover {
     @import "bootstrap-10";
   }
   @import 'bootstrap-inat';
+  [dir="rtl"] & {
+    @import "rtl-bootstrap";
+  }
```

Comment:
Not sure I completely follow. Full duplication of bootstrap.css is not necessary, but to completely adjust it to rtl you need to do around 800 modifications. I can leave it as it is, but I guess that the single page that I will localize wouldn't look like it supposed, because of the cascading from bootstrap.
I also assumed I need to create this file based on looking at your rtl branch where you added  bootstrap-rtl.min.css and added it in the same manner I added it here 
https://github.com/inaturalist/inaturalist/blob/1c41297d6d198d3b37a1ca7371c4a7303c2a0579/app/assets/stylesheets/bootstrap_bundle.scss#L216-L218
I could also replace anything related to direction in bootstrap.css to logical property but then it can potentially have too big of affect on your production. 
Is by overrides you mean working like this? 
https://github.com/inaturalist/inaturalist/blob/1c41297d6d198d3b37a1ca7371c4a7303c2a0579/app/assets/stylesheets/index.scss#L14-L17
or maybe like this?
https://github.com/inaturalist/inaturalist/blob/1c41297d6d198d3b37a1ca7371c4a7303c2a0579/app/assets/stylesheets/users/registrations.scss#L37-L39

Reply:
That’s definitely what I mean by overrides, just curious if you considered that approach and rejected it for some reason. If we don’t use overrides, we should probably make a completely separate rtl bundle, otherwise we’ll serve a lot of unused CSS (rtl csss for ltr people and vice versa). Did you try that and reject it for some reason?